csharp-advanced-topics/SourceGeneratorInCSharp/Generator/Helper/SourceGenerationHelper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Generator.Helper
+{
+    public class SourceGenerationHelper
+    {
+        public const string Attribute = @"
+namespace Generator
+{
+    [System.AttributeUsage(System.AttributeTargets.Class)]
+    public class GenerateServiceAttribute : System.Attribute
+    {
+    }
+}";
```

Comment:
If we use a raw string literal here, we can make the const fit into normal formatting.